.circleci/config.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+
+jobs:
+    build-and-test:
+        macos:
+            xcode: "9.1"
+        steps:
+            - checkout
+            - run:
+                name: Run tests
+                command: set -o pipefail && xcodebuild clean build test -workspace Toshi.xcworkspace -scheme "Tests" -destination 'platform=iOS Simulator,name=iPhone 7' | xcpretty --color && exit ${PIPESTATUS[0]}
```

Comment:
did you mean for this to be toshi instead of teapot?

Reply:
Neither 🧠☠️